api/custom_auth/views.py

```diff
@@ -30,7 +32,21 @@ class CustomAuthTokenLoginWithMFACode(AuthTokenLoginWithMFACode):
     throttle_scope = "mfa_code"
 
 
-class ThrottledUserViewSet(UserViewSet):
+@api_view(["DELETE"])
+@permission_classes([IsAuthenticated])
+def delete_token(request):
```

Comment:
I'm not too sure why we need this endpoint here?

Reply:
By renaming the class ThrottledUserViewSet to FFAdminUserViewSet the endpoint has been moved also.